sycl/test/on-device/sub_group/info.cpp

```diff
@@ -23,9 +23,8 @@ int main() {
   queue Queue;
   device Device = Queue.get_device();
 
-  /* Basic sub-group functionality is supported as part of cl_khr_subgroups
-   * extension or as core OpenCL 2.1 feature. */
-  if (!core_sg_supported(Device)) {
+  // Skip test for SYCL host device.
+  if (!Queue.is_host()) {
```

Comment:
if we skip a test on host, it is better to remove RUN:  env SYCL_DEVICE_TYPE=HOST %t.out and skipping block
to avoid overhead